source/index.html.md

```diff
@@ -798,6 +902,88 @@ Only one 3-D Secure version can be used for a given authorization.
   </dd>
 </dl>
 
+##### Scheme reference to series
+
+If the previous-in-series authorization was made via this API, you should use
+`series[previous][id]` to reference the previous-in-series authorization.
+Otherwise, you must obtain explicit approval from Clearhaus to use the raw
+scheme values grouped in <code>series[previous][mastercard]</code> and
+<code>series[previous][visa]</code>.
+
+The Mastercard specific reference to the series contains the following parts.
```

Comment:
Flip order to start with Visa?

Reply:
I've tried to alphabetize 🤷‍♂️ WDYT?

Reply:
> I've tried to alphabetize 

That was my hunch. :smile: 

Maybe I am colored by my surroundings. But we usually focus on Visa first for several reasons:
* Visa is the biggest of the two schemes globally
* Visa is bigger in Clearhaus' portfolio
* Mastercard is very small in DK, and we still have a DK market bias
* Historically Visa has taken the lead and Mastercard has followed after changing the namingscheme

Am I wrong?

Reply:
You are right AFAICT. (On the last bullet, I guess it depends on the context.) Feel free to push 🙂